src/workspacer.Shared/IConfigContext.cs

```diff
@@ -23,6 +23,8 @@ public interface IConfigContext
 
         IMonitorContainer MonitorContainer { get; set; }
         bool CanMinimizeWindows { get; set; }
+        
+        string UserWorkspacerPath { get; }
```

Comment:
Any reason we need this new property here?

Reply:
Yes, this property is required to retrieve the F# configuration file, as the location is determined in the 'workspacer' project and therefore cannot be used directly.

Reply:
Right. In which case, I guess my main complaint would be about the naming off this property.

What does `UserWorkspacerPath` imply? I don't this name fully conveys that. Is it for the configuration-folder? For the currently running binaries?

I think we need to make this property more self-explanatory :smile:

Reply:
That definitely sounds better to me. I'm all for it.